server/space-service/src/main/java/edu/illinois/ncsa/incore/service/space/controllers/AllocationsController.java

```diff
@@ -0,0 +1,134 @@
+/*******************************************************************************
+ * Copyright (c) 2022 University of Illinois and others.  All rights reserved.
+ * This program and the accompanying materials are made available under the
+ * terms of the Mozilla Public License v2.0 which accompanies this distribution,
+ * and is available at https://www.mozilla.org/en-US/MPL/2.0/
+ *
+ *   Contributors:
+ *   Yong Wook Kim (NCSA) - initial API and implementation
+ *******************************************************************************/
+
+package edu.illinois.ncsa.incore.service.space.controllers;
+
+import edu.illinois.ncsa.incore.common.dao.IGroupAllocationsRepository;
+import edu.illinois.ncsa.incore.common.dao.IUserFinalQuotaRepository;
+import edu.illinois.ncsa.incore.common.exceptions.IncoreHTTPException;
+import edu.illinois.ncsa.incore.common.utils.JsonUtils;
+import io.swagger.annotations.*;
+import org.apache.log4j.Logger;
+import org.json.JSONObject;
+import org.json.simple.parser.ParseException;
+
+import javax.inject.Inject;
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+/**
+ * Created by ywkim on 3/14/2022.
+ */
+
+@SwaggerDefinition(
+    info = @Info(
+        description = "IN-CORE Allocations Service for getting the user's allocation information.",
+        version = "v0.1",
+        title = "IN-CORE v2 Data Service API",
```

Comment:
The title shouldn't be data service api. Same for other status and usage controllers

Reply:
changed, also change usage one